.gitignore

```diff
@@ -1,7 +1,10 @@
 .project
 .settings
 .classpath
-/target
+target
 .idea/
 *.iml
 .vagrant/
+.env
+grafana
+prometheus/data
```

Comment:
Should we ever end up with files in the `grafana` or `prometheus/data` directories? Surely the output is configured to be somewhere more reasonable?

Reply:
This is only for local development, on the live site the docker compose file is generated (https://github.com/ucam-cl-dtg/isaac-monitor) and stores the data in var/lib/graphana|prometheus